app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError(f"{visitor['name']} is not vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError(
+                f"{visitor['name']} has an expired vaccination"
+            )
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError(
+                f"{visitor['name']} didn't wear the mask"
+            )
```

Comment:
According to the checklist, you should avoid using unnecessary `else` statements. Since each `if` block raises an exception, you can remove the `elif` and just use `if` statements for cleaner code.